Helm/opensearch/templates/NOTES.txt

```diff
@@ -0,0 +1,2 @@
+Watch all cluster members come up.
+  $ kubectl get pods --namespace={{ .Release.Namespace }} -l app={{ template "opensearch.uname" . }} -w
```

Comment:
Elasticsearch in general has a ton of components. This may be worth having some info about maybe service names / ports exposed and how people can start throwing documents at ES?

Reply:
I agree we can add some more information for the clusters.